QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151022_002861	The people such as yourself hoping to straighten out the chaos are interesting.{nl}It's a forte that only several amount of people carry in this kingdom.
+QUEST_20151022_002862	Bokors such as ourselves are the clergy that are meant to spread the blessing of goddess to the world.{nl}It's just that the our manner is bit unfamiliar.
+QUEST_20151022_002863	Ahaha. Did you think that the Bokors would have dark personalities?{nl}It seems that misunderstanding of Bokors has spread far.
+QUEST_20151022_002864	There are already many splendid people at Orsha.{nl}I have came down to this place to get closer to entercounter people who's having trouble.
+QUEST_20151022_002865	The meeting with Tesla was a luck of a lifetime.{nl}It'd be great if he would message of his current whereabouts.
+QUEST_20151022_002866	I don't believe it's much of a trouble.{nl}For someone who has dedicated their loyalty to the kingdom, they must be able to sacrifice their life for the subjects of the kingdom in times like these.
+QUEST_20151022_002867	Please don't get surprised.{nl}This fluid is created by Sadhu's ability to help the people of Orsha located around here.
+QUEST_20151022_002868	Paladins such as ourselves, prove the faith of goddess through our strength.{nl}It is our job to protect the people from the monsters or the demons.
+QUEST_20151022_002869	When my uncle finishes his mission, the responsibilites will be handed down to me.{nl}However, through my ability as a Paladin, I want to protect the kingdom and serve the goddess.{nl}If my current ability is mastered, I wish to work towards the mastery of another system.
+QUEST_20151022_002870	Find the ferret's hideout and season the stolen food with the crimson water.{nl}The ferrets hide the stolen foods between the fruit baskets.{nl}
+QUEST_20151022_002871	Ah! The ferrets have stolen beef jerky this time.{nl}The ferrets extremely like beef jerkys. Find the stolen beef jerkeys and make sure to corrupt them.
+QUEST_20151022_002872	Chase after the girl!{nl}She must not meet with the Revelator!
+QUEST_20151022_002873	Finally found you! The ferrets do have their uses.
+QUEST_20151022_002874	There seems to be a fly hidden among us.. Find and eliminate them.
+QUEST_20151022_002875	Halt! You must not be an average fly to make it this far.
+QUEST_20151022_002876	Perfect timing, my body was starting to turn stiff.{nl}Don't you dare think you'll return alive. Pipsqueak!
+QUEST_20151022_002877	Don't you believe this is over.. If you think I'll let you rescue Lada, think again!
+QUEST_20151022_002878	I can't handle over the Kruvina to you!
+QUEST_20151022_002879	Since it's become to this, I will use the power of Kruvina to crush you!
+QUEST_20151022_002880	What are you doing.. Beholder..{nl}This is none of your concern!
+QUEST_20151022_002881	This is Giltine's power.{nl}Not something trifling person like you to desire. 
```

Comment:
Not something trifling person like you to desire. ?

This desirable power is of no importance to (someone like) you. (?)